From 3c420bca8372e1441f977562b61439ff2235003f Mon Sep 17 00:00:00 2001
From: Ronan Abhamon <ronan.abhamon@gmail.com>
Date: Thu, 23 Jun 2022 10:37:07 +0200
Subject: [PATCH 11/61] Remove SR_CACHING capability for many SR types (#24)

SR_CACHING offers the capacity to use IntelliCache, but this
feature is only available using NFS SR.

For more details, the implementation of `_setup_cache` in blktap2.py
uses only an instance of NFSFileVDI for the shared target.

Signed-off-by: Ronan Abhamon <ronan.abhamon@vates.fr>
---
 drivers/CephFSSR.py    | 2 +-
 drivers/GlusterFSSR.py | 2 +-
 drivers/MooseFSSR.py   | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/CephFSSR.py b/drivers/CephFSSR.py
index 415152f7..f334bb30 100644
--- a/drivers/CephFSSR.py
+++ b/drivers/CephFSSR.py
@@ -38,7 +38,7 @@ import vhdutil
 import xs_errors
 from lock import Lock
 
-CAPABILITIES = ["SR_PROBE", "SR_UPDATE", "SR_CACHING",
+CAPABILITIES = ["SR_PROBE", "SR_UPDATE",
                 "VDI_CREATE", "VDI_DELETE", "VDI_ATTACH", "VDI_DETACH",
                 "VDI_UPDATE", "VDI_CLONE", "VDI_SNAPSHOT", "VDI_RESIZE", "VDI_MIRROR",
                 "VDI_GENERATE_CONFIG",
diff --git a/drivers/GlusterFSSR.py b/drivers/GlusterFSSR.py
index 750eb233..290a3312 100644
--- a/drivers/GlusterFSSR.py
+++ b/drivers/GlusterFSSR.py
@@ -35,7 +35,7 @@ import vhdutil
 import xs_errors
 from lock import Lock
 
-CAPABILITIES = ["SR_PROBE", "SR_UPDATE", "SR_CACHING",
+CAPABILITIES = ["SR_PROBE", "SR_UPDATE",
                 "VDI_CREATE", "VDI_DELETE", "VDI_ATTACH", "VDI_DETACH",
                 "VDI_UPDATE", "VDI_CLONE", "VDI_SNAPSHOT", "VDI_RESIZE", "VDI_MIRROR",
                 "VDI_GENERATE_CONFIG",
diff --git a/drivers/MooseFSSR.py b/drivers/MooseFSSR.py
index b2e056af..504e8503 100755
--- a/drivers/MooseFSSR.py
+++ b/drivers/MooseFSSR.py
@@ -39,7 +39,7 @@ import vhdutil
 import xs_errors
 from lock import Lock
 
-CAPABILITIES = ["SR_PROBE", "SR_UPDATE", "SR_CACHING",
+CAPABILITIES = ["SR_PROBE", "SR_UPDATE",
                 "VDI_CREATE", "VDI_DELETE", "VDI_ATTACH", "VDI_DETACH",
                 "VDI_UPDATE", "VDI_CLONE", "VDI_SNAPSHOT", "VDI_RESIZE", "VDI_MIRROR",
                 "VDI_GENERATE_CONFIG",
